ui/transaction_page.go

```diff
@@ -0,0 +1,338 @@
+package ui
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+
```

Comment:
The name of the page file should clearly depict what the page is about. This could be "transaction_details_page.go" and the list of transactions page could be "transactions_page.go"